### PR TITLE
SBP: Installing Fujitsu Enterprise Postgres on SUSE Linux Enterprise Server for IBM Z and LinuxONE - Getting Started

### PR DESCRIPTION
**Pull request by mfriesenegger**

### CODE REVIEW

File: adoc/TRD-fep-vm-sles-s390x-getting-started.adoc [hunk 1]
```diff
@@ -0,0 +1,806 @@
...
+** /dev/mapper/36005076307ffd3b3000000000000013e used for FEP databases (vdb)
+** /dev/mapper/36005076307ffd3b3000000000000013f for FEP transaction logs (vdc)
+** /dev/mapper/36005076307ffd3b30000000000000140 for FEP DB backups (vdd)
+* The installation media is an ISO on the KVM host:
+** /var/lib/libvirt/images/isos/SLE-12-SP5-Server-DVD-s390x-GM-DVD1.iso
+
+
+== Configuration of the virtual machine
+
+The goal of this chapter is to successfully install SLES 12 SP5.  To accomplish this the VM is started and the autoyast installation begins automatically.
```
Comment (cjschroder) [inline adoc/TRD-fep-vm-sles-s390x-getting-started.adoc:137]: autoyast -> AutoYAST

File: adoc/TRD-fep-vm-sles-s390x-getting-started.adoc [hunk 2]
```diff
@@ -0,0 +1,806 @@
...
+
+
+[start=2]
+. Restart the virtual server by using the following command to complete phase 2 of the automated installation:
++
+----
+ virsh start fep --console
+----
++
+NOTE: You can safely ignore any postfix warnings or errors that appear in the console.
```
Comment (cjschroder) [inline adoc/TRD-fep-vm-sles-s390x-getting-started.adoc:222]: postfix -> Postfix

File: adoc/TRD-fep-vm-sles-s390x-getting-started.adoc [hunk 3]
```diff
@@ -0,0 +1,806 @@
...
+ ./WebAdminSetup
+
+. Accept the default port numbers for the:
+  * Web Server (27515)
+  * Internal port number for WebAdmin (27516)
+
+. Answer "yes" for WebAdmin autostart
+
+
+NOTE: The systemctl service name for FEP 12 is FSEPWAD_120_WA_64.service
```
Comment (cjschroder) [inline adoc/TRD-fep-vm-sles-s390x-getting-started.adoc:277]: FSEPWAD looks like a name teenagers insult each other with :)

File: adoc/TRD-fep-vm-sles-s390x-getting-started.adoc [hunk 4]
```diff
@@ -0,0 +1,806 @@
...
+** Four IBM Integrated Facility for Linux (IFL) processors
+** 12 GB storage (RAM)
+** Two Fibre Channel Protocol (FCP) adapters for multipath
+** Four 25 GB Small Computer System Interface (SCSI) Logical Unit Number (LUN):
+*** One LUN is used for the KVM host operating system
+*** Three LUNs are used to store the databases, transaction logs and database backups
+** One or two Open System Adapters (OSA):
+*** OSA 1 (eth0) is for SSH connections to the KVM host and for VEPA-based devices for VMs
+*** OSA 2 (eth1) is optional, but can be used for VEPA-based devices for VMs
+* A workstation with a web browser and ssh client will be needed.
```
Comment (cjschroder) [inline adoc/TRD-fep-vm-sles-s390x-getting-started.adoc:80]: web -> Web, ssh -> SSH

**Review by cjschroder — CHANGES_REQUESTED** [review]
This looks good, Mike, well-organized and clear. I found only a few small typos.

**Comment by cjschroder** [comment]
I see I missed a few typos. Arg. :frowning_face: 
Today is a German holiday, I will leave this for Meike to review when she returns.

**Review by cjschroder — APPROVED** [review]

**Comment by chabowski** [comment]
Thanks @cjschroder and @mfriesenegger - I'll take if from here. Will merge the PR now and have final check /fix typos.
Thanks again!

**Comment by chabowski** [comment]
Final (??) PDF version of the paper for your review:
[TRD-fep-vm-sles-s390x-getting-started_color_en.pdf](https://github.com/SUSE/suse-best-practices/files/6540772/TRD-fep-vm-sles-s390x-getting-started_color_en.pdf)

**Comment by mfriesenegger** [comment]
The PDF looks like it has all of the final edits.

**Comment by chabowski** [comment]
New PDF with non-working link :-) as discussed: 
[TRD-fep-vm-sles-s390x-getting-started_color_en.pdf](https://github.com/SUSE/suse-best-practices/files/6541216/TRD-fep-vm-sles-s390x-getting-started_color_en.pdf)

**Pull request merged**